### PR TITLE
igwejk/even up the defaults across ghas features

### PR DESCRIPTION
**Pull request by igwejk**
- Remove .vscode/launch.json in favour of npm scripts
- WIP

### CODE REVIEW

**Comment by NickLiffen** [comment]
@igwejk lmk when this is ready for review 🙇

File: src/utils/predicates.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  {
+    primaryLanguage: { name: repositoryPrimaryLanguage = "no-language" },
+  }: RepositoryNode = {} as RepositoryNode,
+  languageToCheck: string = process.env.LANGUAGE_TO_CHECK as string
+): boolean {
+  if (!languageToCheck) {
+    return true;
+  }
+
+  return new RegExp(`^${languageToCheck}$`, "i").test(
```
[the remark this answers is not in the document]
Comment (igwejk) [reply src/utils/predicates.ts:51]: Fixed.

**Comment by igwejk** [comment]
@NickLiffen it's ready now

**Comment by igwejk** [comment]
cc @Adarsh1925

**Review by NickLiffen — APPROVED** [review]
Really nice work ✨

**Pull request merged**